src/components/pastAnalysis/seriesTable.tsx

```diff
@@ -26,45 +28,66 @@ export const isLargestNumber = (num: number | null | undefined, numArray: Map<st
-  });
+  if (classifications.length) {
+    classifications.forEach((value: string) => {
+      titles.push({ title: (<span><br /><span className='classificationText'>{value}</span></span>) }); // Adding the column titles for each analysis
+    });
+  } else {
+    titles.push({ title: (<span><br /><span className='classificationText'>{"Results"}</span></span>) }); // Adding column for studies containing only failed analyses
+  }
 
   titles.push({ title: (<span className='classificationText'><span>Geographic<br />Severity</span></span>) },
-              { title: (<span className='classificationText'><span>Opacity<br />Extent</span></span>) },
-              { title: (<span></span>) });
+    { title: (<span className='classificationText'><span>Opacity<br />Extent</span></span>) },
+    { title: (<span></span>) });
 
   const columns = titles;
 
   const rows = analysisList.map((analysis: ISeries, index: number) => {
+    const isAnalysisValid = !!analysis.classifications.size;
     let analysisCells: any = [
       { title: (analysis.imageUrl ? <div><b><img src={analysis.imageUrl} className="thumbnail" /></b></div> : <div><PreviewNotAvailable /></div>) },
       { title: (<div><b>{analysis.imageName.split('/').pop()}</b></div>) }
     ];
 
-    analysis.classifications.forEach((value: number, key: string) => { // Dynamically displaying each prediction class
-      analysisCells.push({
-        title: (<PredictionCircle key={key}
-          largeCircle={isLargestNumber(value, analysis.classifications)}
-          predictionNumber={value} />)});
-    });
-  
+    if (isAnalysisValid) { // If this Series' analysis was successful, read in its analysis results
+      analysis.classifications.forEach((value: number, key: string) => { // Dynamically displaying each prediction class
+        analysisCells.push({
+          title: (<PredictionCircle key={key}
+            largeCircle={isLargestNumber(value, analysis.classifications)}
+            predictionNumber={value} />)
+        });
+      });
+    } else if (classifications?.length) { // If this Series' analysis was unsuccessful, but the list of classes is known, display 'N/A' for each classification result
+      analysisCells.push(...classifications.map(() => ({
+        title: ('N/A')
+      })));
+    } else { // If this Series' analysis was unsuccessful, and the list of classes is unknown
       analysisCells.push({
-        title: `${analysis.geographic ? `${analysis.geographic.severity}` : 'N/A'}`
-      }, {
-        title: `${analysis.opacity ? `${analysis.opacity.extentScore}` : 'N/A'}`
-      }, {
-        title: (<Button variant="secondary" onClick={() => viewImage(index)} isDisabled={isProcessing}>View</Button>)
+        title: ('Results not available')
       });
+    }
+
+    analysisCells.push({
+      title: `${analysis.geographic ? `${analysis.geographic.severity}` : 'N/A'}`
+    }, {
+      title: `${analysis.opacity ? `${analysis.opacity.extentScore}` : 'N/A'}`
```

Comment:
I know that this code was from previous commits but since there are changes to this line thought it might be a good opportunity to refactor it as well.

What do you think about this approach? @janakitti @colinywu 
```suggestion
      title: analysis?.geographic?.severity || 'N/A'
    }, {
      title: analysis?.opacity?.extentScore || 'N/A'
```

In my opinion it looks a bit cleaner and shorter as well as doesn't try to access object properties twice. Open for discussion.